diff --git a/node_modules/expo-router/build/views/Sitemap.js b/node_modules/expo-router/build/views/Sitemap.js
--- a/node_modules/expo-router/build/views/Sitemap.js
+++ b/node_modules/expo-router/build/views/Sitemap.js
@@ -82,16 +82,16 @@
             ], children: [(0, jsx_runtime_1.jsxs)(react_native_1.View, { style: { flexDirection: 'row', alignItems: 'center' }, children: [leftIcon, (0, jsx_runtime_1.jsx)(react_native_1.Text, { style: styles.filename, children: filename })] }), (0, jsx_runtime_1.jsxs)(react_native_1.View, { style: { flexDirection: 'row', alignItems: 'center' }, children: [!!info && (0, jsx_runtime_1.jsx)(react_native_1.Text, { style: [styles.virtual, { marginRight: 8 }], children: info }), rightIcon] })] })) }));
 }
 function FileIcon() {
-    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, source: require('expo-router/assets/file.png') });
+    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, resizeMode: "contain", source: require('expo-router/assets/file.png') });
 }
 function PkgIcon() {
-    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, source: require('expo-router/assets/pkg.png') });
+    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, resizeMode: "contain", source: require('expo-router/assets/pkg.png') });
 }
 function ForwardIcon() {
-    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, source: require('expo-router/assets/forward.png') });
+    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, resizeMode: "contain", source: require('expo-router/assets/forward.png') });
 }
 function SitemapIcon() {
-    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, source: require('expo-router/assets/sitemap.png') });
+    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { style: styles.image, resizeMode: "contain", source: require('expo-router/assets/sitemap.png') });
 }
 function ArrowIcon({ rotation = 0 }) {
     return ((0, jsx_runtime_1.jsx)(react_native_1.Image, { style: [
@@ -99,7 +99,7 @@
             {
                 transform: [{ rotate: `${rotation}deg` }],
             },
-        ], source: require('expo-router/assets/arrow_down.png') }));
+        ], resizeMode: "contain", source: require('expo-router/assets/arrow_down.png') }));
 }
 function SystemInfo() {
     const getHermesVersion = () => {
@@ -205,7 +205,7 @@
     },
     filename: { color: 'white', fontSize: 20, marginLeft: 12 },
     virtual: { textAlign: 'right', color: 'white' },
-    image: { width: 24, height: 24, resizeMode: 'contain', opacity: 0.6 },
+    image: { width: 24, height: 24, opacity: 0.6 },
     headerIcon: {
         width: 40,
         height: 40,
diff --git a/node_modules/expo-router/build/views/Toast.js b/node_modules/expo-router/build/views/Toast.js
--- a/node_modules/expo-router/build/views/Toast.js
+++ b/node_modules/expo-router/build/views/Toast.js
@@ -51,14 +51,14 @@
                     }),
                     opacity: value,
                 },
-            ], children: [!warning && (0, jsx_runtime_1.jsx)(react_native_1.ActivityIndicator, { color: "white" }), warning && (0, jsx_runtime_1.jsx)(react_native_1.Image, { source: require('expo-router/assets/error.png'), style: styles.icon }), (0, jsx_runtime_1.jsxs)(react_native_1.View, { style: { marginLeft: 8 }, children: [(0, jsx_runtime_1.jsx)(react_native_1.Text, { style: styles.text, children: children }), filenamePretty && (0, jsx_runtime_1.jsx)(react_native_1.Text, { style: styles.filename, children: filenamePretty })] })] }) }));
+            ], children: [!warning && (0, jsx_runtime_1.jsx)(react_native_1.ActivityIndicator, { color: "white" }), warning && (0, jsx_runtime_1.jsx)(react_native_1.Image, { source: require('expo-router/assets/error.png'), resizeMode: "contain", style: styles.icon }), (0, jsx_runtime_1.jsxs)(react_native_1.View, { style: { marginLeft: 8 }, children: [(0, jsx_runtime_1.jsx)(react_native_1.Text, { style: styles.text, children: children }), filenamePretty && (0, jsx_runtime_1.jsx)(react_native_1.Text, { style: styles.filename, children: filenamePretty })] })] }) }));
 }
 const styles = react_native_1.StyleSheet.create({
     container: {
         backgroundColor: 'transparent',
         flex: 1,
     },
-    icon: { width: 20, height: 20, resizeMode: 'contain' },
+    icon: { width: 20, height: 20 },
     toast: {
         alignItems: 'center',
         borderWidth: 1,
diff --git a/node_modules/expo-router/build/views/Unmatched.js b/node_modules/expo-router/build/views/Unmatched.js
--- a/node_modules/expo-router/build/views/Unmatched.js
+++ b/node_modules/expo-router/build/views/Unmatched.js
@@ -104,7 +104,7 @@
                                 ], children: "Sitemap" })) }) })] })] }));
 }
 function NotFoundAsset() {
-    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { source: require('expo-router/assets/unmatched.png'), style: styles.image });
+    return (0, jsx_runtime_1.jsx)(react_native_1.Image, { source: require('expo-router/assets/unmatched.png'), resizeMode: "contain", style: styles.image });
 }
 const styles = react_native_1.StyleSheet.create({
     container: {
@@ -118,7 +118,6 @@
     image: {
         width: 270,
         height: 168,
-        resizeMode: 'contain',
         marginBottom: 28,
     },
     title: {
